monitor/utils/api.py

```diff
@@ -20,10 +20,10 @@
 
 from monitor import exceptions as ex
 from monitor.utils import serializer as u_serializer
-from monitor.utils.logger import *
```

Comment:
This is a unused import? If yes, please comment this modification in the description of this PR.